content/docs/setup/kubernetes/mesh-expansion/index.md

```diff
@@ -5,272 +5,280 @@ weight: 50
 
-{{< text bash >}}
-$ export GCP_OPTS="--zone MY_ZONE --project MY_PROJECT"
-$ export SERVICE_NAMESPACE=vm
-{{< /text >}}
+1.  Install the debian package with the Envoy sidecar:
 
-If you are running on a GCE VM, run
-
-{{< text bash >}}
-$ @install/tools/setupMeshEx.sh@ gceMachineSetup VM_NAME
-{{< /text >}}
+    {{< text bash >}}
 
-Otherwise, run
+    $ curl -L https://storage.googleapis.com/istio-release/releases/1.0.0/deb/istio-sidecar.deb > istio-sidecar.deb
+    $ dpkg -i istio-sidecar.deb
 
-{{< text bash >}}
-$ @install/tools/setupMeshEx.sh@ machineSetup VM_NAME
-{{< /text >}}
+    {{< /text >}}
 
-GCE provides better user experience since node agent can always relies on
-GCE metadata instance document to authenticate to Citadel. For everything
-else, e.g., on-prem or raw VM, we have to bootstrap a key/cert as credential,
-which typically has a limited lifetime. And when the cert expires, you have to
-rerun the above command.
+2.  Copy the cluster.env and *.pem files that you created in the previous section to the VM.
 
-Or the equivalent manual steps:
+3.  Add the IP address of the Istio gateway (which we found in the previous section) to `/etc/hosts` or to the DNS server. In our example we'll use `/etc/hosts` as it is the easiest way to get things working. The following is an example of updating an `/etc/hosts` file with the Istio gateway address:
 
------- Manual setup steps begin ------
+    {{< text bash>}}
 
-* Copy the configuration files and Istio Debian files to each machine joining the cluster.
-Save the files as `/etc/dnsmasq.d/kubedns` and `/var/lib/istio/envoy/cluster.env`.
+    $ echo "35.232.112.158 istio-citadel istio-pilot istio-pilot.istio-system" >> /etc/hosts
 
-*   Configure and verify DNS settings. This may require installing `dnsmasq` and either
-adding it to `/etc/resolv.conf` directly or via DHCP scripts. To verify, check that the VM can resolve
-names and connect to pilot, for example:
+    {{< /text >}}
 
-    On the VM/external host:
+4.  Install `root-cert.pem`, `key.pem` and `cert-chain.pem` under `/etc/certs`.
 
     {{< text bash >}}
-    $ host istio-pilot.istio-system
-    {{< /text >}}
 
-    Example generated message:
+    $ sudo mkdir /etc/certs
+    $ sudo cp {root-cert.pem,cert-chain.pem,key.pem} /etc/certs
 
-    {{< text plain >}}
-    $ istio-pilot.istio-system has address 10.150.0.6
     {{< /text >}}
 
-    Check that you can resolve cluster IPs. The actual address will depend on your deployment.
+6.  Install `cluster.env` under `/var/lib/istio/envoy`.
 
     {{< text bash >}}
-    $ host istio-pilot.istio-system.svc.cluster.local.
-    {{< /text >}}
 
-    Example generated message:
+    $ sudo cp cluster.env /var/lib/istio/envoy
 
-    {{< text plain >}}
-    istio-pilot.istio-system.svc.cluster.local has address 10.63.247.248
     {{< /text >}}
 
-    Check istio-ingress similarly:
+5.  Transfer ownership of the files in `/etc/certs/` and `/var/lib/istio/envoy` to the Istio proxy.
```

Comment:
To distinguish more clearly between what is a file and what is a folder, I suggest you add the nouns after their names:
```suggestion
5.  Transfer ownership of the files in the `/etc/certs/` and `/var/lib/istio/envoy` folders to the Istio proxy.
```